src/main/resources/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStep/help-propagate.html

```diff
@@ -2,4 +2,5 @@
     If set, then if the downstream build is anything but successful (blue ball), this step fails.
```

Comment:
nit: I'd tweak this line instead and delete the new line you added.

```suggestion
     If enabled (it is enabled by default), then if the downstream build is anything but successful (blue ball), this step fails.
```